apps/labs/posts/pep-517-build-system-popularity.md

```diff
@@ -0,0 +1,584 @@
+---
+title: 'PEP 517 build system popularity'
+published: January 24, 2025
+authors: [michal-gorny]
+description: 'A closer look at non-elementary group-by aggregations'
```

Comment:
I recognise this sentence from somewhere 😉

Reply:
Haha, sorry, forgot to update it :-).